demos/security-compare-asset-allocations/demo.js

```diff
@@ -0,0 +1,122 @@
+
+async function displaySecurityDetails (postmanJSON) {
+    const ids = ['F000015CH2', 'F0000143IP'],
+        idNames = {
+            'F000015CH2': 
+                'Schroder International Selection Fund Global Emerging Market Opportunities',
+            'F0000143IP': 'Fidelity Funds - Asia Pacific Opportunities Fund I-Acc-USD'
+        }
+
+
+    const connector = new HighchartsConnectors.Morningstar.SecurityCompareConnector({
+        postman: {
+            environmentJSON: postmanJSON
+        },
+        security: {
+            ids,
+            idType: 'MSID'
+        },
+        converter: {
+            type: 'AssetAllocations'
+        }
+    });
+
+    await connector.load();
+
+    const typeMapping = {
+        '1': 'Stocks',
+        '2': 'Bonds',
+        '3': 'Cash',
+        '4': 'Other Instruments',
+        '99': 'Unclassified'
+    };
+
+    const result = ids.map(id => 
+        connector.table.getRowObjects()
+            .map(item => ({
+                name: typeMapping[item['AssetAllocations_Type_' + id]],
+                y: item['AssetAllocations_MorningstarEUR3_N_' + id]
+            }))
+            .filter(entry => entry.y !== 0) // Filter out entries where y is 0
```

Comment:
I think we have to keep all values, otherwise we have mismatching colors in series (compare cash): 

![Zrzut ekranu 2025-02-18 o 13 07 16](https://github.com/user-attachments/assets/12dfdbe5-70bd-4cc4-836b-4e43f1cc5995)